app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = current_user.comments.new(post_params)
+    @comment.post_id = params[:post_id]
+    respond_to do |format|
+      if @comment.save
+        format.html { redirect_to user_post_path(@comment.author_id, @comment.post_id) }
+      else
+        format.html { render :new }
+      end
+    end
```

Comment:
- [ ] As suggested by the previous reviewer, please create a `view/comments/new.html.erb` view file which will help you accept inputs in order to create a comment

Reply:
Thanks fixed